corehq/apps/locations/util.py

```diff
@@ -205,7 +205,7 @@ def get_headers(self):
         ])
         for loc_type in self.location_types:
             additional_headers = []
-            additional_headers.extend(self._prefix_headers('data', (f.slug for f in self.data_model.fields)))
```

Comment:
`_prefix_headers` does a lot of stuff in a roundabout way, but essentially all it's used for is to prefix the column headers with `"data: "`.  Unfortunately it also sorted things alphabetically, which meant that headers and column data didn't line up.

Reply:
should the other usage of `_prefix_headers` also be modified if it alphabetizes them?